### ISSUE TITLE
Unkown error in running sedef

### ISSUE THREAD

**Issue by mossishahi** [vpc-ccg/sedef#11]
Hi

I would like to run Sedef on a fragmented draft genome of planaria but I face these errors in different cases:

Firstly I launched the command 
`./sedef.sh -o test1sedef -j 16 -t translation.fa DjScaff_fn.fa`
that faced error but produced the translation.fa

the log:

```
Indexing ../DjScaff_fnl20141213.fa...                                                                                                                         [369/1844]
Translating ...
🍁  🐚    SEDEF 1.1-23-ge21e2e8; arguments:  (SSE4.1) sedef translate ../DjScaff_fnl20141213.fa translation.fa
Translating ../DjScaff_fnl20141213.fa to translation.fa
Chromosome   1, size  100,032,929
Chromosome   2, size  100,104,382
Chromosome   3, size  100,055,355
Chromosome   4, size  100,031,847
Chromosome   5, size  100,048,904
Chromosome   6, size  100,040,151
Chromosome   7, size  100,031,787
Chromosome   8, size  100,017,402
Chromosome   9, size  100,017,611
Chromosome  10, size  100,005,020
Chromosome  11, size  100,005,679
Chromosome  12, size  100,000,939
Chromosome  13, size  100,001,380
Chromosome  14, size  100,001,406
Chromosome  15, size  100,000,970
Chromosome  16, size   85,287,642
Done with 16 chromosomes
Translating took 36.5s
Translation time: 0:36.69 (1691980 MB, user 12.00)
************************************************************************
Running SD seeding...
Unknown option: will-cite
Usage:

parallel [options] [command [arguments]] < list_of_arguments
parallel [options] [command [arguments]] (::: arguments|:::: argfile(s))...
cat ... | parallel --pipe [options] [command [arguments]]

-j n            Run n jobs in parallel
-k              Keep same order
-X              Multiple arguments with context replace
--colsep regexp Split input on regexp for positional replacements
{} {.} {/} {/.} {#} {%} {= perl code =} Replacement strings
{3} {3.} {3/} {3/.} {=3 perl code =}    Positional replacement strings
With --plus:    {} = {+/}/{/} = {.}.{+.} = {+/}/{/.}.{+.} = {..}.{+..} =
                {+/}/{/..}.{+..} = {...}.{+...} = {+/}/{/...}.{+...}

-S sshlogin     Example: foo@server.example.com
--slf ..        Use ~/.parallel/sshloginfile as the list of sshlogins
--trc {}.bar    Shorthand for --transfer --return {}.bar --cleanup
--onall         Run the given command with argument on all sshlogins
--nonall        Run the given command with no arguments on all sshlogins

--pipe          Split stdin (standard input) to multiple jobs.
--recend str    Record end separator for --pipe.
--recstart str  Record start separator for --pipe.

See 'man parallel' for details

When using programs that use GNU Parallel to process data for publication please cite:

O. Tange (2011): GNU Parallel - The Command-Line Power Tool,
;login: The USENIX Magazine, February 2011:42-47.

Or you can get GNU Parallel without this requirement by paying 10000 EUR.
Command exited with non-zero status 255
Seeding time: 0:00.37
SD seeding done: done running 22 jobs!
grep: test1sedef/log/seeds/*.log: No such file or directory
Error: launched 22 jobs but completed only 0 jobs; exiting...
```
then

I attempted to launch the main sedef using this command:
`sedef -o output1 -j 16 translation.fa`

and again the it resulted in an error:
```
Start: ‫سه‌شنبه ۲۱ مه ۱۹، ساعت ۰۲:۴۷:۱۶ (+0430)‬
SEDEF: FASTA=translation.fa; output=output1; jobs=16; force=n
************************************************************************
Running SD seeding...
Unknown option: will-cite
Usage:

parallel [options] [command [arguments]] < list_of_arguments
parallel [options] [command [arguments]] (::: arguments|:::: argfile(s))...
cat ... | parallel --pipe [options] [command [arguments]]

-j n            Run n jobs in parallel
-k              Keep same order
-X              Multiple arguments with context replace
--colsep regexp Split input on regexp for positional replacements
****
{} {.} {/} {/.} {#} {%} {= perl code =} Replacement strings
{3} {3.} {3/} {3/.} {=3 perl code =}    Positional replacement strings
With --plus:    {} = {+/}/{/} = {.}.{+.} = {+/}/{/.}.{+.} = {..}.{+..} =
                {+/}/{/..}.{+..} = {...}.{+...} = {+/}/{/...}.{+...}

-S sshlogin     Example: foo@server.example.com
--slf ..        Use ~/.parallel/sshloginfile as the list of sshlogins
--trc {}.bar    Shorthand for --transfer --return {}.bar --cleanup
--onall         Run the given command with argument on all sshlogins
--nonall        Run the given command with no arguments on all sshlogins

--pipe          Split stdin (standard input) to multiple jobs.
--recend str    Record end separator for --pipe.
--recstart str  Record start separator for --pipe.

See 'man parallel' for details

When using programs that use GNU Parallel to process data for publication please cite:

O. Tange (2011): GNU Parallel - The Command-Line Power Tool,
;login: The USENIX Magazine, February 2011:42-47.

Or you can get GNU Parallel without this requirement by paying 10000 EUR.
Command exited with non-zero status 255
Seeding time: 0:00.20
SD seeding done: done running 12 jobs!
grep: output1/log/seeds/*.log: No such file or directory
Error: launched 12 jobs but completed only 0 jobs; exiting...
```

long story short that I tried to run it using differen mode of commands and trying different cases but didn't resulted in any success.

Thanks

**Comment by inumanag** [comment]
Argh, GNU parallel and its citation parameter...
Can you try removing `--will-cite` from sedef.sh and running it again? Thanks

**Comment by inumanag** [comment]
Closing due to inactivity--- please open another issue if this still persists.
